html/index.php

```diff
@@ -1,7 +1,72 @@
 <?php
 require '../vendor/autoload.php';
+require_once 'HTML/Table.php';
 
+const DOCROOT = '/var/www/';
 
+$dbmanager = new dbManager(DOCROOT . 'railerdb.sqlite3');
```

Comment:
Might be overkill but you could use a package such as `pimple` to manage this sort of thing.